src/codegen/codegen_c_visitor.cpp

```diff
@@ -1251,6 +1251,9 @@ std::string CodegenCVisitor::compute_method_name(BlockType type) const {
     if (type == BlockType::Initial) {
         return method_name(naming::NRN_INIT_METHOD);
     }
+    if (type == BlockType::Destructor) {
+        return method_name(naming::NRN_DEST_METHOD);
```

Comment:
Can I do one another `nitpick` ? short form `DEST` is interpreted as `destination`. I suggest `NRN_DESTRUCT_METHOD` to be more clear?